usaspending_api/etl/elasticsearch_loader_helpers/index_config.py

```diff
@@ -57,27 +69,24 @@ def swap_aliases(client, config):
         old_indexes = list(client.indices.get_alias("*", alias_patterns).keys())
         for old_index in old_indexes:
             client.indices.delete_alias(old_index, "_all")
-            logger.info(format_log(f"Removing aliases from '{old_index}'", process="ES Alias"))
+            logger.info(format_log(f"Removing aliases from '{old_index}'", action="ES Alias"))
     except Exception:
-        logger.exception(f"No aliases found for {alias_patterns}", process="ES Alias")
+        logger.exception(f"No aliases found for {alias_patterns}", action="ES Alias")
 
     create_aliases(client, config)
 
     try:
         if old_indexes:
             client.indices.delete(index=old_indexes, ignore_unavailable=False)
-            logger.info(format_log(f"Deleted index(es) '{old_indexes}'", process="ES Alias"))
+            logger.info(format_log(f"Deleted index(es) '{old_indexes}'", action="ES Alias"))
     except Exception:
-        logger.exception(f"Unable to delete indexes: {old_indexes}", process="ES Alias")
+        logger.exception(f"Unable to delete indexes: {old_indexes}", action="ES Alias")
 
 
 def toggle_refresh_off(client, index):
     client.indices.put_settings({"refresh_interval": "-1"}, index)
-    message = (
-        f'Set "refresh_interval": "-1" to turn auto refresh off during incremental load. Manual refreshes will '
-        f"occur for each batch completion."
-    )
-    logger.info(format_log(message, process="ES Settings"))
+    message = (f'Set "refresh_interval": "-1" to turn auto refresh off')
```

Comment:
You should indicate that refreshes will be done for each batch. Otherwise someone just reading the logs will get the impression that as data is changing throughout the incremental load, it is _not_ surfacing for queries. 

That, or 
- add another log statement somewhere saying that refreshes are done per batch
- add a log statement to each refresh for each batch (probably too overly verbose)

Reply:
See also reply here: https://github.com/fedspendingtransparency/usaspending-api/pull/2809#discussion_r501042467

Reply:
> add another log statement somewhere saying that refreshes are done per batch

I'm still opposed to manual refreshes after each batch, see [comment](https://github.com/fedspendingtransparency/usaspending-api/pull/2809#discussion_r501049749)